tests/layer_tests/tensorflow_tests/test_tf_FloorDiv.py

```diff
@@ -17,9 +16,8 @@ def create_add_placeholder_const_net(self, x_shape, dtype, ir_version, use_new_f
         # Create the graph and model
         with tf.compat.v1.Session() as sess:
             x = tf.compat.v1.placeholder(dtype, x_shape, 'Input')
-            constant_value = np.array(-10).astype(dtype)
+            constant_value = np.array(-256).astype(dtype)
             y = tf.constant(constant_value)
-            x = tf.raw_ops.Abs(x=x)
```

Comment:
this abs is not necessary here, and it causes problems when GPU inference is checked
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/aac283d5-8a43-49eb-ac87-1c9f1944ca3c)